packages/backend/src/api/routes/status.ts

```diff
@@ -0,0 +1,15 @@
+import Router from '@koa/router'
+
+import { StatusController } from '../controllers/StatusController'
+
+export function createStatusRouter(statusController: StatusController): Router {
+  const router = new Router()
+
+  router.get('/status/discovery', async (ctx): Promise<void> => {
```

Comment:
Can be just `/status` imo

Reply:
Yeah, but in the future we can have like gnosis safe status or what not